### PR TITLE
Modal Demo

### PR DESCRIPTION
**Pull request by AFogarty72**
Added modals rendering elements inside, as well as added primary modal
type. Requires blur admin update to apply changes. **

### CODE REVIEW

**Comment by AFogarty72** [comment]
closes #52

**Comment by AFogarty72** [comment]
screen shot 2016-07-27 at 8 07 46 am

File: src/page/modal-demo.js [hunk 1]
```diff
@@ -22,6 +24,12 @@ export class ModalDemo extends React.Component {
     this.setState({ [modalName]: true });
   }
 
+  onSwitchChange(index) {
+    let switches = this.state.switches;
```
Comment (jwdotjs) [inline src/page/modal-demo.js]: We should use `_.assign` here so we don't ever overwrite this.state.variableName directly

File: src/page/modal-demo.js [hunk 2]
```diff
@@ -33,6 +42,111 @@ export class ModalDemo extends React.Component {
...
+
+        <Modal type='warning' title='With Switches' buttonText='Save' isOpen={this.state.customizedModal5} onClose={e => this.onCloseModal('customizedModal5')}>
+          <Row>
+            <Col>
+              Notify me when...
+            </Col>
+          </Row>
+          <Row>
+            <Col>
+              <Col grow={false}>
```
Comment (jwdotjs) [inline src/page/modal-demo.js]: I don't think it works to have a `Col` inside a `Col` but it is ok to have a `Row`, then a `Col` with another set of Rows/Cols inside it

**Pull request merged**